generators/client/templates/angular/src/main/webapp/app/layouts/breadcrumb/_breadcrumb.component.html

```diff
@@ -0,0 +1,14 @@
+<ol class="breadcrumb">
+    <li class="breadcrumb-item">
+        <a [hidden]="breadcrumbs.length === 0" routerLink=""><i class="fa fa-home"></i></a>
+        <span [hidden]="breadcrumbs.length > 0"><i class="fa fa-home"></i></span>
+    </li>
+    <li class="breadcrumb-item" [ngClass]="{ 'active': last }" *ngFor="let breadcrumb of breadcrumbs;  let last = last">
+        <a [hidden]="last" [routerLink]="[breadcrumb.url]">{{breadcrumb.label | translate}}</a>
+        <%_ if (enableTranslation) { _%>
```

Comment:
this is not needed here. The `| translate` will be stripped away (if needed) with regex.